simcon_project/users/views/view_response.py

```diff
@@ -12,9 +12,14 @@
 @user_passes_test(is_authenticated)
 def view_response(request, pk):
     response = get_object_or_404(TemplateResponse, pk=pk)
-
+    if request.method == "POST":
+        response_table = TemplateResponse.objects.all()
+        items_to_delete = request.POST.getlist('delete_items')
+        response_table.filter(pk__in=items_to_delete).delete()
+        return HttpResponseRedirect(reverse('researcher-view'))
```

Comment:
I totally should have mentioned this before but there's a few other places on the site where we've used delete buttons that trigger modals to confirm deletion. Seems like we should try to be consistent about it. The settings page for the admin has a good example of what I mean. I can help implement if you'd like